packages/backend/src/services/UserService.ts

```diff
@@ -1180,9 +1180,16 @@ export class UserService {
             return {
                 showOptions: [openIdIssuer],
                 forceRedirect: true,
-                redirectUri: getRedirectUri(openIdIssuer),
+                redirectUri: new URL(
+                    `/api/v1${getRedirectUri(openIdIssuer)}`,
```

Comment:
full url, not related with rate limit. I pushed here by mistake, we should merge it anyway